gulpfile.js

```diff
@@ -296,10 +296,7 @@ gulp.task('mocha', function (done) {
         error = err;
       })
       .on('end', function () {
-        // When the tests are done, disconnect mongoose and pass the error state back to gulp
-        mongoose.disconnect(function () {
-          done(error);
-        });
+        mongooseService.disconnect(done);
```

Comment:
Yea, good points. It did occur to me, for a moment, about the `error` needing to be passed to the done. Then I think I just overlooked it. I'll change it to pass the Mocha error to the `done` callback. I guess we can just log the error to the console, when the `disconnect` fails; it would be a rare error in this case.

The name `error` seems fine to me. It's clear what's going on since it's right there.